### PR TITLE
teams remaining pages and functionality changed in to hotwire/stimulu…

### PR DESCRIPTION
**Pull request by sadrisya**
…s framework

### CODE REVIEW

File: app/javascript/controllers/team_user_info_controller.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    const buttonId = event.currentTarget.id;
+    if (buttonId === "de-activate-yes") {
+      console.log("deactivate");
+      const confirmationModal = this.element.querySelector(
+        ".deactivate-confirmation"
+      );
+      confirmationModal.classList.add("hidden");
+      this.deActiveTarget.classList.remove("hidden");
+      this.activeTarget.classList.add("hidden");
+      this.deActiveMenuTarget.textContent = "Active";
```
Comment (deepakkumarnd) [inline app/javascript/controllers/team_user_info_controller.js:37]: Text content has to be Activate and Deactivate

File: app/views/dashboards/_index_team_desktop.html.erb [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
-            <li class="flex space-x-1">
-               <a href="#" class="">TATA-Hotels</a>
-               <span class="icon icon-simple-arrow-right h-4 w-4"></span>
-            </li>
-            <li class="flex space-x-1">
-               <a href="#" class="">Top Leaders</a>
-               <span class="icon icon-simple-arrow-right h-4 w-4"></span>
-            </li>
-         </ol>
+<main class="" >
```
Comment (deepakkumarnd) [inline app/views/dashboards/_index_team_desktop.html.erb:2]: main tag is already part of the layout

**Review by deepakkumarnd — CHANGES_REQUESTED** [review]

File: app/javascript/controllers/team_user_info_controller.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["deActive", "active", "deActiveMenu"];
```
[the remark this answers is not in the document]
Comment (sadrisya) [reply app/javascript/controllers/team_user_info_controller.js]: changes are done

File: app/javascript/controllers/team_user_info_controller.js (hunk 1 above)
Comment (sadrisya) [reply app/javascript/controllers/team_user_info_controller.js]: texts changed

File: app/views/dashboards/_index_team_desktop.html.erb (hunk 2 above)
Comment (sadrisya) [reply app/views/dashboards/_index_team_desktop.html.erb]: removed main tag

**Review by deepakkumarnd — APPROVED** [review]

**Pull request merged**